osf/models/institution.py

```diff
@@ -103,13 +106,22 @@ def banner_path(self):
             return None
 
     def update_search(self):
-        from website.search.search import update_institution
+        from website.search.search import update_institution, update_node
         from website.search.exceptions import SearchUnavailableError
+
         try:
             update_institution(self)
         except SearchUnavailableError as e:
             logger.exception(e)
 
+        saved_fields = self.get_dirty_fields()
+        if saved_fields and bool(self.pk):
+            for node in self.nodes.filter(is_deleted=False):
+                try:
+                    update_node(node, async=False)
```

Comment:
Woops, I had a typo in my above comment (now corrected). I think we only need to update the institution in ES if it's been changed, i.e. `saved_fields` is truthy.